App/Lettuce/Sources/Lettuce/Extensions/String+Extensions.swift

```diff
@@ -0,0 +1,8 @@
+
+import Foundation
+
+public extension String {
+    static func localized(_ key: String) -> String {
```

Comment:
It looks like this already exists? https://developer.apple.com/documentation/xcode/localizing-and-varying-text-with-a-string-catalog#Localize-your-apps-text You can check the rest of this document for hints if this extension is needed or not :)

Reply:
It's for shortening the syntax - I want to avoid passing `tableName: "LettuceLocalizable"`. and `bundle: .module` each time